src/App/Properties/Strings.en.resx

```diff
@@ -631,4 +631,28 @@ The new version is {1} (You currently have version {2})</value>
   <data name="ThreatProtection" xml:space="preserve">
     <value>Threat Protection</value>
   </data>
+  <data name="DnsMonitoring" xml:space="preserve">
+    <value>Dns Monitoring</value>
+  </data>
+  <data name="EnableDoubleHop" xml:space="preserve">
+    <value>Enable Double Hop (Only available with OpenVPN and WireGuard)</value>
+  </data>
+  <data name="SelectEntryLocation" xml:space="preserve">
+    <value>Select an entry location to route your VPN connection through:</value>
+  </data>
+  <data name="TrafficOptimizer" xml:space="preserve">
+    <value>Traffic Optimizer</value>
+  </data>
+  <data name="EnableTrafficOptimizer" xml:space="preserve">
+    <value>Enable traffic optimizer</value>
+  </data>
+  <data name="DoubleHop" xml:space="preserve">
+    <value>Double Hop</value>
+  </data>
+  <data name="MaxThreeApplication" xml:space="preserve">
+    <value>Please note that a maximum of 3 applications can only be optimized.</value>
```

Comment:
Please change it to

`Please note that only up to 3 applications can be optimized at a time.`

Reply:
Resolved